server/src/services/library.service.ts

```diff
@@ -250,15 +243,14 @@ export class LibraryService {
     return mapLibrary(library);
   }
 
-  private async scanAssets(libraryId: string, assetPaths: string[], ownerId: string, force = false) {
+  private async scanAssets(libraryId: string, assetPaths: string[], ownerId: string) {
```

Comment:
Maybe we should rename this to syncFiles? Also maybe `syncFiles(assetPaths: string, { libraryId, ownerId }: { libraryId: string, ownerId: string })` would look better